_meta/beat.yml

```diff
@@ -4,13 +4,13 @@
 
 apm-server:
   # Defines the host and port the server is listening on
-  host: "localhost:8080"
 
-  #server.max_unzipped_size:
-  #server.max_header_bytes:
-  #server.read_timeout: 2s
-  #server.write_timeout: 2s
+  #host: "localhost:8080"
```

Comment:
Thanks Nicolas, everyone needs to be aware of this, indeed. 2 things:
+ removed `server` inner namespace (eg: `apm-server.secret_token` instead of `apm-server.server.secret_token`
+ commented out `host`, as it is defaulted already in the codebase.